src/test/scala/CacheKeyTestSpec.scala

```diff
@@ -0,0 +1,114 @@
+import org.scalatest._
+
+import scala.sys.process._
+class CacheKeyTest {
+  private var cacheKey1 = Option("cacheKey1")
+  private var cacheKey2 = Option("cacheKey2")
+  private var gitCommitsMade = 0
+  private var dependenciesAdded = 0
+
+  def cleanUp(): Unit = {
```

Comment:
This is never called. It definitely should be.

I would suggest adding the `before` and `after` calls to the test, else you risk leaving this in a corrupt state.